CONTRIBUTING.md

```diff
@@ -1,122 +1,58 @@
-# Contributing to Zeebe
-
-* [Build Zeebe from source](#build-zeebe-from-source)
```

Comment:
I would propose to keep "build from source" at one of the first steps / at the top of the page, since this a very foundation to contribute.

Reply:
@MaxTru, I moved it up higher, but not right at the top. My thinking is that the doc is in order -- first prerequisites for everyone (CLA, CoC) to more specific. Since some may be looking for our Issue guidelines and you don't need to build from source for that (correct me if I'm wrong)-- I put those higher. But let me know if you feel strongly that the build process should be higher up!